src/createAnimatedComponent/createAnimatedComponent.tsx

```diff
@@ -254,7 +254,45 @@ export function createAnimatedComponent(
           has('workletEventHandler', prop) &&
           prop.workletEventHandler instanceof WorkletEventHandler
         ) {
-          prop.workletEventHandler.unregisterFromEvents();
+          prop.workletEventHandler.unregisterFromEvents(this._viewTag);
+        }
+      }
+    }
+
+    _updateNativeEvents(
```

Comment:
Does it make sense to move this method where `_reattachNativeEvents` used to be for easier review and tracking changes in git blame?

Reply:
I think this function wasn't complex enough to really care about it in git blame, but we can revert the name change if you insist.